### PR TITLE
feat: update network id to goerli

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

update create temporary dns for goerli network

## Changes

- update network id to 5

### CODE REVIEW

File: src/implementations/config/create.ts [hunk 1]
```diff
@@ -1,19 +1,19 @@
+import { utils, v2, v3 } from "@govtechsg/open-attestation";
 import fs from "fs";
 import path from "path";
 import { info } from "signale";
-import { readFile } from "../../implementations/utils/disk";
-import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
 import { CreateConfigCommand } from "../../commands/config/config.type";
-import { Dns } from "./types";
+import { handler as createTemporaryDns } from "../../commands/dns/txt-record/create";
+import { readFile } from "../../implementations/utils/disk";
```
Comment (superical) [inline src/implementations/config/create.ts:7]: This import could be just this:

```suggestion
import { readFile } from "../utils/disk";
```

**Review by superical — APPROVED** [review]
Looks good! Just one tiny little suggestion for one of the imports.
Interesting that the imports are sorted nicely! Was it prettier that did the sorting?🤔 

The import suggestion is pretty minor. So I will approve it first so that you can merge as soon as you are ready.

File: src/implementations/config/create.ts (hunk 1 above)
Comment (isaackps) [reply src/implementations/config/create.ts:7]: ok

**Comment by isaackps** [comment]
> Looks good! Just one tiny little suggestion for one of the imports. Interesting that the imports are sorted nicely! Was it prettier that did the sorting?🤔
> 
> The import suggestion is pretty minor. So I will approve it first so that you can merge as soon as you are ready.

hahha yup its the prettier that did the sorting.

**Pull request merged**